packages/next-codemod/lib/utils.ts

```diff
@@ -116,4 +116,10 @@ export const TRANSFORMER_INQUIRER_CHOICES = [
     value: 'app-dir-runtime-config-experimental-edge',
     version: '15.0.0-canary.179',
   },
+  {
+    title:
+      'Transform `revalidateTag` and `revalidatePath` to `expireTag` and `expirePath`',
+    value: 'revalidate-to-expire',
+    version: '15.0.4-canary.12',
```

Comment:
@ijjk Forgot to add the prompt choice. 😅
Is it a good choice to target the canary versions for the newly added codemods?

Reply:
Yea the first canary we changed it over should be good

Reply:
In the past, we used the maximum of codemod introduced and change introduced since people got confused that the codemod was suggested even though they already made the change. That was for very old changes though that were already out for multiple stable releases. 

It probably doesn't matter since it's just a few canaries but good to keep in mind in the future.